gst/nnstreamer/tensor_query/tensor_query_serversrc.c

```diff
@@ -189,6 +190,152 @@ gst_tensor_query_serversrc_finalize (GObject * object)
   G_OBJECT_CLASS (parent_class)->finalize (object);
 }
 
+/**
+ * @brief nnstreamer-edge event callback.
+ */
+static int
+_nns_edge_event_cb (nns_edge_event_h event_h, void *user_data)
+{
+  nns_edge_event_e event_type;
+  int ret = NNS_EDGE_ERROR_NONE;
+
+  GstTensorQueryServerSrc *src = (GstTensorQueryServerSrc *) user_data;
+  if (0 != nns_edge_event_get_type (event_h, &event_type)) {
+    nns_loge ("Failed to get event type!");
+    return NNS_EDGE_ERROR_UNKNOWN;
+  }
+
+  switch (event_type) {
+    case NNS_EDGE_EVENT_NEW_DATA_RECEIVED:
+    {
+      nns_edge_data_h data;
+
+      nns_edge_event_parse_new_data (event_h, &data);
```

Comment:
plz consider to handle internal error case,
```
ret = nns_edge_event_parse_new_data (event_h, &data);
if (ret != NNS_EDGE_ERROR_NONE)
  g_async_queue_push (src->msg_queue, data);
```

Reply:
Updated.